src/assertions/descendantsCount.js

```diff
@@ -0,0 +1,22 @@
+export default function descendantsCount ({ wrapper, markup, arg1, arg2=1, sig }) {
+  this.assert(
+      wrapper.hasDescendantsCount(arg1, arg2),
+      () => 'expected ' + sig + ' to have ' + arg2 + ' descendants #{exp} ' + markup(),
+      () => 'expected ' + sig + ' not to have ' + arg2 + ' descendants #{exp} ' + markup(),
+      arg1
+  )
+}
+
+function convenienceHelper(count) {
+  return function ({ wrapper, markup, arg1, sig }) {
+    const arg2 = count;
+    descendantsCount.call(this, { wrapper, markup, arg1, arg2, sig});
+  };
+}
+
+export const descendantsCountWrappers = {
```

Comment:
i'm not sure i see the value in adding these "convenience" wrappers. what's the logic for having "four"? why not "five"? It'd be much clearer imo to just require `hasDescendantsCount` for all calls, and it creates less diff churn if you have to go between 4 and 5, for example.